cvat-canvas/src/typescript/shared.ts

```diff
@@ -41,7 +41,7 @@ export interface DrawnState {
     occluded?: boolean;
     hidden?: boolean;
     lock: boolean;
-    source: 'AUTO' | 'SEMI-AUTO' | 'MANUAL';
+    source: 'AUTO' | 'SEMI-AUTO' | 'MANUAL' | 'FILE';
```

Comment:
I believe we need to update ``computeNewSource`` function in ``cvat-core/src/annotations-objects.ts``. 
It is responsible for getting new source after an object was modified somehow.